docs/source/faq.ipynb

```diff
@@ -66,6 +66,31 @@
     "```"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### What is the optimal file format to use with `vaex`?\n",
+    "\n",
+    "What is \"optimal\" may dependant on what one is trying to achieve. A quick summary would be:\n",
+    "\n",
+    "- For performance: [HDF5](https://en.wikipedia.org/wiki/Hierarchical_Data_Format);\n",
+    "- For interoperability: [Apache Arrow](https://arrow.apache.org/);\n",
+    "- For optimizing disk space: [Apache Parquet](https://parquet.apache.org/).\n",
+    "\n",
+    "`vaex` shines when the data is in a memory-mappable file format, namely HDF5, Apache Arrow, or [FITS](https://en.wikipedia.org/wiki/FITS). We say a file is memory mappable if it has the same structure in memory, as it has on disk. \n",
+    "\n",
+    "In principle, HDF5 and Arrow should give the same performance. For files that would fit into memry the performance between the two is the same. For single files that are larger available RAM, our tests show that HDF5 gives faster performance. What \"faster\" means will likely depend on your system, quantity and type of data.\n",
+    "\n",
+    "If your data is spread amongst multiple files that are concatenated on the fly, the performance between HDF5 and Arrow is expected to be the same. Our test show better performance when all of the data is contained in a single file, compared to multipe file.\n",
+    "\n",
+    "The Arrow file format allows seemless interoperability with other eco-systems. If your use-case requires sharing data with other eco-systems, e.g. Java, the Arrow file format is the way to.\n",
+    "\n",
+    "`vaex` also supports Parquet. Parquet is compressed, and thus not memory mappable. There is always a performance penalty when using Parquet, since the data needs to be decompressed before it is used. Parquet however allows lazy reading of the data, which can be decompressed on the fly. Thus `vaex` canvan easily work with Parquet files that are larger than RAM. We recommend using Parquet when one wants to save disk space. It can be also convenient when reading from i/o sources, like Cloud storage for example.\n",
```

Comment:
```suggestion
    "`vaex` also supports Parquet. Parquet is compressed, and memory mappable brings no advantage. There is always a performance penalty when using Parquet, since the data needs to be decompressed before it is used. Parquet however allows lazy reading of the data, which can be decompressed on the fly. Thus `vaex` canvan easily work with Parquet files that are larger than RAM. We recommend using Parquet when one wants to save disk space. It can be also convenient when reading from slow i/o sources, like spinner harddrives or Cloud storage for example. Note that using [df.materialize](api.html#vaex.dataframe.DataFrame.materialize) one can get the same performance as hdf5 or arrow files at the cost of memory (or disk space)\n",
```